lib/java/persistance/src/main/java/org/enso/persist/Persistance.java

```diff
@@ -0,0 +1,218 @@
+package org.enso.persist;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.function.Function;
+
+import static org.enso.persist.PerUtils.raise;
+
+/** Central persistance class. Use static {@link
```

Comment:
Use:
```bash
enso$ find lib/java/persistance/src/main/java/ | grep java$ | xargs ~/bin/graalvm-21/bin/javadoc -d target/javadoc/ --snippet-path lib/java/persistance/src/test/java/
```
to generate `target/javadoc/index.html` and view this documentation in proper form.